bash-completion/test.json

```diff
@@ -6,6 +6,9 @@
   "versionSpecific": false,
   "type": "bash",
   "cleanup": true,
+  "skipWhen": [
+    "ci" // bash completion script not installed
```

Comment:
Is `ci` the right term to mean "non-packaged builds"? It is possible (and a distant goal of mine) that we can package up nightly commit of .NET (now VMR) as an RPM package and test it.

Reply:
I'm open to any other name here. This is the one that came to mind.

Reply:
Some other options:

- `sdk-tarball`
- `tarball`
- `not-installed`
- `not-packaged`

Reply:
Which one do you prefer?

Reply:
I used the name `ci` to mark the tests that are skipped in our ci. And with a comment why that test is skipped.
So with one name I'm describing the traits of the ci.
I can change it to `not-packaged` if you prefer that, though it means something different than `ci` then. 🙃

Reply:
I've renamed it to `vmr-ci`.